src/rgw/rgw_sync_trace.cc

```diff
@@ -15,6 +15,8 @@
 
 
 #define dout_context g_ceph_context
+#define dout_subsys ceph_subsys_rgw
+
```

Comment:
I'd be inclined to make this a `static constexpr` but that's just nitpicking.

Reply:
Nice idea. So I did that with the files I needed to modify and a few other RGW .cc files as a reminder to do it more broadly.